src/tests/StreamerTest.cpp

```diff
@@ -228,6 +239,37 @@ class StreamerProcessTimingTest : public ::testing::Test {
   StreamerStandIn TestStreamer;
 };
 
+TEST_F(StreamerProcessTimingTest, MessageHasNoTimestamp) {
```

Comment:
Name of the test should also contain an indication of the expected behavior.

Reply:
:+1:

Reply:
:+1: